redaxo/src/core/lib/sql/sql.php

```diff
@@ -415,14 +415,23 @@ public function setQuery($query, array $params = [], array $options = [])
         }
 
         try {
-            $this->stmt = rex_timer::measure(__METHOD__, static function () use ($pdo, $query) {
-                return $pdo->query($query);
+            $this->stmt = rex_timer::measure(__METHOD__, function () use ($pdo, $query) {
+                error_clear_last();
+
+                if (false !== $stmt = @$pdo->query($query)) {
```

Comment:
Eigentlich dachte ich, bei uns könnte da kein false rauskommen, da wir PDO im Exception-Mode nutzen.
Aber just heute habe ich in Slack gelernt, dass es doch Fälle geben kann: https://friendsofredaxo.slack.com/archives/C1BAXLN2F/p1605084523101100
Daher versuche ich nun die Fälle abzufangen und die Warnung in einen Error zu wandeln.